test/templates/default/usingGradle/gradle/liquibase.gradle

```diff
@@ -0,0 +1,55 @@
+import org.gradle.internal.os.OperatingSystem
```

Comment:
I have no idea where this comes from, I created an application using Gradle and JHipster v3.3, then paste the resulting file. Is there a problem with it ?